GitUI/FormSignatureInfo.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace GitUI
+{
+    public partial class FormSignatureInfo : Form
+    {
+        public FormSignatureInfo()
+        {
+            InitializeComponent();
+        }
+
+        public void setGpgMessage(string message)
```

Comment:
Please use PascalCase for methods.
Not a big fan of this name, I would probably call it `ShowGpgMessage(string)`

Reply:
ok for PascalCase, but I think it's more a "set" than a "show" method.

Reply:
we're showing the message to the user, aren't we? :)

Reply:
Changed to PascalCase